streams/site.yaml

```diff
@@ -0,0 +1,41 @@
+#######################################################################
+## Copyright (c) 2018 Contributors to the Eclipse Foundation
+##
+## See the NOTICE file(s) distributed with this work for additional
+## information regarding copyright ownership.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+#######################################################################
+%YAML 1.2
+---
+documentation:
+  - title: Configuration for Microprofile
```

Comment:
This file needs to be updated to reflect reactive stream spec.

Reply:
Thanks, I knew there was something I missed to do with the spec.